recipe/meta.yaml

```diff
@@ -136,33 +114,25 @@ requirements:
     # Otherwise build and host might end up with two different versions
     # If they do, merge_build_host will fail.
     - readline                           # [linux]
+    - llvm                               # [osx]
   host:
     # For pkg-config gl, fontconfig + freetype
     - pthread-stubs                      # [linux]
     - dbus                               # [linux]
     - fontconfig                         # [linux]
     - freetype                           # [linux]
-    - gst-plugins-base                   # [linux]
-    - gstreamer                          # [linux]
+    - glib                               # [not arm64]
+    - gst-plugins-base                   # [not arm64]
+    - gstreamer                          # [not arm64]
```

Comment:
Why not arm64?

Reply:
It is not available in `arm64`: https://anaconda.org/conda-forge/gstreamer/

Reply:
Can you send a PR?